packages/components/src/tooltip/src/TooltipTrigger.tsx

```diff
@@ -159,6 +157,9 @@ export function InnerTooltipTrigger({
     ));
 
     const tooltipMarkup = augmentElement(tooltip, {
+        arrowProps: {
```

Comment:
could of simply passe arrowRef, but popover passes "arrowProps" so i felt like it would be better if they both had the same name